### ISSUE TITLE
About the concatenation of self.null_k and k.

### ISSUE THREAD

**Issue by gitprwang** [lucidrains/memory-compressed-attention#3]
Thanks for the implementation of memory compressed attention. I found it very useful when building my own language models.

I got an error in line58 of memory_compressed_attention.py when the batch size is set as numbers other than 1. The error results from the concatenation of self.null_k and k. When batch size is not 1, the two metrices have different size at dim 0. I think line 58 and 59 should be,
```
        k = torch.cat((self.null_k.broadcast_to(b, 1, d), k), dim=1)
        v = torch.cat((self.null_v.broadcast_to(b, 1, d), v), dim=1)
```

**Comment by lucidrains** [comment]
@gitprwang yes, thank you, should be addressed in 0.0.7

are you seeing a signal using this technique?

**Comment by gitprwang** [comment]
@lucidrains Everything goes well when using `broadcast_to`. No warning or other trivial message in terminal.

**Comment by gitprwang** [comment]
> @lucidrains Everything goes well when using `broadcast_to`. No warning or other trivial message in terminal.

My pytorch version is 1.13

**Comment by lucidrains** [comment]
@gitprwang oh, are you still seeing error messages with 0.0.7? i used [.expand instead](https://github.com/lucidrains/memory-compressed-attention/blob/master/memory_compressed_attention/memory_compressed_attention.py#L58)

**Comment by lucidrains** [comment]
think it should be fixed, feel free to reopen if not!
